scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,671 @@
+            class="btn btn-outline-secondary form-control" target="_blank">Submit to ClinVar</a>
+        {% endif %}
+        {% if variant.clinvar_clinsig %}
+          <a href="{{ url_for('overview.clinvar_submissions', institute_id=institute._id) }}" class="btn btn-outline-secondary form-control">Modify clinvar submission</a>
+        {% endif %}
+      </div>
+      {% if evaluations %}
+        <div class="list-group mt-3">
+          {% for evaluation in evaluations %}
+            {{ acmg_classification_item(evaluation) }}
+          {% endfor %}
+        </div>
+      {% endif %}
+    </div>
+  </div>
+{% endmacro %}
+
+
+{% macro panel_quality() %}
+  <div class="card panel-default">
+    <div class="panel-heading">Quality</div>
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">QUAL:
+        <span class="font-weight-bold">
+          {% if variant.quality %}
+            {{ variant.quality|int }}
+          {% else %}
+            -
+          {% endif %}
+        </span>
+        <li class="list-group-item">{{ mappability(variant) }} </li>
+    </ul>
+  </div>
+{% endmacro %}
+
+
+{% macro mappability(variant) %}
+  {% set superdups_fracmatches = [] %}
+  {% for gene in variant.genes %}
+    {% for transcript in gene.transcripts %}
+      {% if transcript.superdups_fracmatch %}
+        {% for superdup in transcript.superdups_fracmatch %}
+          {% do superdups_fracmatches.append(superdup) %}
+        {% endfor %}
+      {% endif %}
+    {% endfor %}
+  {% endfor %}
+  {% set superdups_fracmatches = superdups_fracmatches|sort %}
+  {% if superdups_fracmatches %}
+    Mappability: <span>mapping to {{superdups_fracmatches|length}} segm. dups. (min matching:{{ superdups_fracmatches|first|float|round(3) }}, max matching:{{ superdups_fracmatches|last|float|round(3) }})</span>
+  {% else %}
+    <p class="card-text">Mappability: -</p>
+  {% endif %}
+{% endmacro %}
+
+
+{% macro panel_comment() %}
+  <div class="card panel-default">
+    {{ comments_panel(institute, case, current_user, variant.comments, variant_id=variant._id) }}
+  </div>
+{% endmacro %}
+
+
+{% macro panel_basics() %}
+      <table class="table table-sm">
+        <tbody>
+          <tr>
+          </tr>
+          {% if variant.str_repid %}
+          <tr>
+            <td colspan="2">
+              Repeat locus
+              <span><strong>{{variant.str_repid}}</strong></span>
+            </td>
+          </tr>
+          {% endif %}
+        </tbody>
+      </table>
+      <ul class="list-group">
+        <div class="list-group-item">
+          <strong>Gene panels:</strong><br>
+          {% for panel_id in variant.panels %}
+            <a href="{{ url_for('panels.panel', panel_id=panel_id) }}">{{ panel_id }}</a>&nbsp;&nbsp;
+          {% endfor %}
+        </div>
+      </ul>
+{% endmacro %}
+
+
+{% macro panel_summary() %}
```

Comment:
Also this panel looks exactly the same as the summary panel of SNVs, I think it should be shared.